### PR TITLE
PVC Tenants link

### PR DESCRIPTION
**Pull request by adfost**
Link to Tenants from PVC page

Screen Shot 2021-07-06 at 12 28 59 PM

### CODE REVIEW

**Review by Alevsk — APPROVED** [review]

**Review by dvaldivia — CHANGES_REQUESTED** [review]
these need to link to the Tenant who owns the PVC

File: portal-ui/src/screens/Console/Storage/Storage.tsx [hunk 1]
```diff
@@ -65,7 +65,16 @@ const Storage = ({ classes, match }: IStorageProps) => {
 
   return (
     <Fragment>
-      <PageHeader label={"Storage"} />
+      <PageHeader
+        label={
+          <Fragment>
+            Storage:
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/Storage.tsx:71]: Please add an space next to Storage:

```suggestion
            Storage:&nbsp;
```

File: portal-ui/src/screens/Console/Storage/Storage.tsx [hunk 2]
```diff
@@ -65,7 +65,16 @@ const Storage = ({ classes, match }: IStorageProps) => {
...
+            <Link to={"/tenants"} className={classes.breadcrumLink}>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/Storage.tsx:72]: Please make this link to point to the tenant who has this PVC attached

File: portal-ui/src/screens/Console/Storage/Storage.tsx [hunk 3]
```diff
@@ -65,7 +65,16 @@ const Storage = ({ classes, match }: IStorageProps) => {
...
+              Tenants
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/Storage.tsx:73]: I this is better to place the name of the tenant here instead of a `Tenants` label

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**